### PR TITLE
site: Add TOC to configuration page, show page title in <title>, add spacing below sidebar

### PR DESCRIPTION
**Pull request by AndiDog**
See separate commits

### CODE REVIEW

**Comment by aojea** [comment]
/ok-to-test
it renders ok, but my knowledge in this area is close to zero :smile: 
/assign @BenTheElder

**Comment by AndiDog** [comment]
Rebased to resolve merge conflict

File: site/layouts/partials/fancymarkdown.html [hunk 1]
```diff
@@ -1 +1,11 @@
-{{ . | replaceRE "(<h[1-9] id=\"([^\"]+)\".+)(</h[1-9]+>)" `${1}<a href="#${2}" class="hanchor" ariaLabel="Anchor"> 🔗&#xFE0E;</a> ${3}` | safeHTML }}
\ No newline at end of file
+{{ if not (eq .Site.Title .Params.title) }}
+<h1>{{ .Params.title }}</h1>
+{{ end }}
+
```
Comment (BenTheElder) [inline site/layouts/partials/fancymarkdown.html:4]: we're injecting blank lines if these aren't set?

Comment (AndiDog) [reply site/layouts/partials/fancymarkdown.html:4]: I guess so, but have hopefully covered all main pages. Not sure how to make it mandatory with Hugo.

Comment (BenTheElder) [reply site/layouts/partials/fancymarkdown.html:4]: we can put the blank lines inside the blocks before the `{{ end }}`

**Comment by BenTheElder** [comment]
thank you! and thank you for your patience!

**Pull request merged**